src/frontend/komponenter/Fagsak/Behandlingsresultat/EndretUtbetalingAndelSkjema.tsx

```diff
@@ -129,6 +131,16 @@ const EndretUtbetalingAndelSkjema: React.FunctionComponent<IEndretUtbetalingAnde
         );
     };
 
+    function inneholderAlleredeUtbetaltBegrunnelser() {
```

Comment:
Dette er veldig småpirk, du får se om du ønsker å gjøre dette eller ikke: Jeg ser at vi sjekker to ganger om `skjema.felter.vedtaksbegrunnelser.verdi` har en verdi. Du kan eventuelt sende inn `skjema.felter.vedtaksbegrunnelser.verdi` som et parameter i funksjonen `inneholderAlleredeUtbetaltBegrunnelser()`, altså `inneholderAlleredeUtbetaltBegrunnelser(vedtaksbegrunnelser: Begrunnelse[])`. Siden du allerede har gjort sjekken om `skjema.felter.vedtaksbegrunnelser.verdi` finnes her trengs det ikke å gjøres to ganger. 😊